package/yast2.changes

```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Fri May 20 08:09:04 UTC 2016 - mchandras@suse.de
+
+- Split SuSEFirewall and SuSEFirewallServices classes to separate files
+  as recommended by the Yast coding style (fate#318356, gh#yast/yast-yast2#471)
```

Comment:
Nitpicking: Please try to keep `max 80 characters on oneline` rule in changes file